website/docs/installation/nix.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 4
+---
+# Nix
+
+If you are using Nix, **git-cliff** can be installed from the [official latest staple channel][channel-link].
+
+## Using nix-shell
+
+To temporarily install git-cliff in a shell environment, run:
+
+```sh
+nix-shell -p git-cliff
+```
+
+## Using nix-env
+
+To install git-cliff permanently, use:
+
+```sh
+nix-env -iA nixpkgs.git-cliff
+```
+
+> [!WARNING]
+> Using nix-env permanently modifies a local profile of installed packages.
+> This must be updated and maintained by the user in the same way as with a traditional package manager, 
+> foregoing many of the benefits that make Nix uniquely powerful. 
+> Using nix-shell or a NixOS configuration is recommended instead.
+
+
+## The new CLI
+
+If you're using the new experimental CLI, you can use any of the following:
+
+### `nix run
```

Comment:
wops, we have typo here, i forget to close this (`)

Reply:
fixed in main